docs/developing/languages/ruby/installation.md

```diff
@@ -16,52 +12,131 @@ for several reasons:
   should be assumed that everything under `/System/Library` is part of macOS and
   is administered by Apple. Files in those directories should not be modified.
 
-- [macOS will no longer include Ruby by default in future releases](https://developer.apple.com/documentation/macos_release_notes/macos_catalina_10_15_beta_9_release_notes).
+## For quick scripts
 
-## Approaches
+It's possible to install Ruby using `homebrew`, however, it's difficult to install precise versions. For one-off uses, though, it's a reasonable approach.
```

Comment:
Honestly, I wouldn't bother. I tried it for a while and it always ended up being a mess. I've had a lot better luck installing via asdf and then setting a global ruby and python for one-offs.